src/js/components/Pagination/StyledPageControl.js

```diff
@@ -30,8 +67,30 @@ const sizeStyle = props => {
 };
 
 export const StyledPaginationButton = styled(Button)`
-  padding: 4px;
-  border-radius: 4px;
+  ${props => {
+    const buttonStyle =
+      typeof props.kind === 'string'
+        ? buildButtonStyle(props.theme.button, props.kind)
+        : props.kind;
+    const adjustedPadding = calcAdjustedPadding(buttonStyle, defaultStyles.pad);
+    let buttonState = 'default';
```

Comment:
I get the usage of `default` here as the default state but could it cause confusion with `default` as a button kind? Wondering if there is any other term we can use. soft opinion.

Reply:
Good call. I will adjust.

Reply:
Addressed.